Bottom: 77bd2beb9d8e7ede1f55bebef3572781352800db
Top:    728ee98372ae9bb11c1e1e360c278ade5dcee591
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-08-07 15:40:19 +0800

Refresh of squash

---

diff --git a/net/core/filter.c b/net/core/filter.c
index f126cad..9c78254 100644
--- a/net/core/filter.c
+++ b/net/core/filter.c
@@ -3315,14 +3315,13 @@ EXPORT_SYMBOL_GPL(xdp_do_redirect);
 int xdp_do_rx_handler(struct net_device *dev, struct xdp_buff *xdp)
 {
 	rx_handler_xdp_func_t *rx_handler_xdp;
-	int ret = 0;
 
 	rx_handler_xdp = rcu_dereference(dev->rx_handler_xdp);
 	if (rx_handler_xdp) {
 		switch(rx_handler_xdp(xdp)) {
 		case RX_HANDLER_CONSUMED:
 			return 0;
-		case RX_HANDLER_ANOHTER:
+		case RX_HANDLER_PASS:
 			return -ENOTSUPP;
 		default:
 			return -EFAULT;
